app/models/payroll_run.rb

```diff
@@ -34,6 +34,10 @@ def download_available?
     download_triggered? && Time.zone.now - downloaded_at < DOWNLOAD_FILE_TIMEOUT.seconds
   end
 
+  def confirmation_report_uploaded?
```

Comment:
Now we have this method I reckon there are a couple of other places we could be using it (see `payroll_runs/index` and `payroll_runs/show`).